doc/source/cookbook/fits_radio_cubes.ipynb

```diff
@@ -450,7 +450,7 @@
    },
    "outputs": [],
    "source": [
-    "print (box_reg.quantities.extrema(\"temperature\"))"
+    "print (box_reg.quantities.extrema(\"gas\", (\"temperature\")))"
```

Comment:
Shouldn't it be
```suggestion
    "print (box_reg.quantities.extrema((\"gas\", \"temperature\")))"
```

Reply:
You're right.  I figured I'd mess up a missing character in all of these changes.  Good catch.